examples/spiders/form_action.py

```diff
@@ -0,0 +1,46 @@
+import scrapy
+from scrapypuppeteer import PuppeteerRequest, PuppeteerScreenshotResponse
+from scrapypuppeteer.actions import Screenshot, FormAction
+import base64
+
+class FormActionSpider(scrapy.Spider):
+    name = 'form_action'
+    start_urls = ["https://www.roboform.com/filling-test-all-fields"]
+
+    def start_requests(self):
```

Comment:
When started with `PUPPETEER_LOCAL` it gave many extra lines in the logs.
Should be fixed in other PR.

Reply:
I don't really understand where this problem is coming from...